Library/Homebrew/keg_relocate.rb

```diff
@@ -97,7 +99,12 @@ def detect_cxx_stdlibs(_options = {})
   end
 
   def each_unique_file_matching(string)
-    Utils.popen_read("/usr/bin/fgrep", "-lr", string, to_s) do |io|
+    bsd = `/usr/bin/fgrep -V`.include?("BSD grep")
```

Comment:
I think I'd have to copy the body of the method into both os/mac/keg_relocate.rb and os/linux/keg_relocate.rb and just change the arguments that are being passed to grep. Is that right? It feels repetitive.

Why not backticks, out of curiosity? Consistency or something more subtle?